New tools being added based on request from IBM on Jan 11, 2024 (#27)
EasyDynamics/oscal.io: _tools/compliance-trestle-agile-authoring.md

---
title: Compliance Trestle Agile Authoring
author: IBM
license: Apache-2.0 license
link: https://github.com/oscal-compass/compliance-trestle-agile-authoring
---

> A repo that comprises documentation and tutorial for OSCAL-oriented agile authoring pipelines. This repo is meant to provide an OSCAL authoring collabrative gitops workflow.

Reviewer: ```suggestion
A repo comprising of documentation and tutorials for OSCAL-oriented Agile authoring pipelines through a collaborative GitHub workflow.

```
